hooks/post_checkout

```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
-# downloads a local copy of qemu on docker-hub build machines
-curl -L https://github.com/balena-io/qemu/releases/download/v3.0.0%2Bresin/qemu-3.0.0+resin-arm.tar.gz | tar zxvf - -C . && mv qemu-3.0.0+resin-arm/qemu-arm-static .
```

Comment:
This isn't required. The pro_build hook should be enough.

Reply:
I know, and it worked locally before and the hooks worked just fine. You know what's really weird? The autobuild right now works when done from the `arm-autobuild-test`, which gave me the confidence to merge it into master. When merged, it failed.

Reply:
Very strange. I don't have access to the logs and I am to lazy to setup my own repo because Docker Hub always takes ages to build. If you are not happy with it I could try to fix it if I got some logs. On the other hand, if it works right now just leave it.

Reply:
Do you have access to this URL? https://hub.docker.com/repository/registry-1.docker.io/tzahi12345/youtubedl-material/builds/79b1ddfb-88e2-49c6-86d6-a9721c56c12d

That's where the most recent logs are. If you can't access, then give me your Docker ID and I'll add you as a collaborator to the Docker Hub page.

Reply:
The URL just redirects me to https://hub.docker.com/r/tzahi12345/youtubedl-material.

My Docker HUB ID is https://hub.docker.com/u/supersandro2000.

Reply:
I went ahead and added you as a collaborator, hopefully that lets you see the build logs. Let me know if it does!

Reply:
Nope it doesn't. Also nothing changed on the repo for me. No new buttons or anything. Maybe you did a typo by accident?

Reply:
I think it's because my repo is under a user and not an organization, which means I can't give admin access (only allow contributors to push and pull, but not see logs. I know, it's weird). I created an organization and added you as an `owner` for testing. I hooked into the autobuild setup and there's one running now: https://hub.docker.com/repository/registry-1.docker.io/ytdlmaterial/youtubedl-material/builds/55f85406-e745-45dd-9ec0-c847c9282813

You should be able to access that without issue!

Reply:
That works! Not sure how you wanna go from here though.

Reply:
It was just to show you the logs of what failed during the building process, this is where it failed:

```
---> Running in 76cf60c4e1db
[91mstandard_init_linux.go:211: exec user process caused "exec format error"
[0m
Removing intermediate container 76cf60c4e1db
The command '/bin/sh -c addgroup -S $USER -g $GID && adduser -D -S $USER -G $USER -u $UID' returned a non-zero code: 1
```

Reply:
> exec user process caused "exec format error"

Usually means it tries to execute something that is not existing or in $PATH.

Reply:
Interesting, maybe it should be /bin/bash instead of /bin/sh? Or is it saying addgroup isn't a command?

>Docker HUB is just so slow that debugging is not a pleasant experience.

You're right, it's very unpleasant lol. Basically what I do is build the arm version locally, and if that works, test it out on Docker Hub in a branch, and if that works, merge to master. That's basically what I did with this PR; it failed once I merged to master.

Takes a bit of time, but it's ultimately necessary for the ARM version, at least for now. If you have a better suggestion for autobuilding for ARM, let me know!

Reply:
> Interesting, maybe it should be /bin/bash instead of /bin/sh?

They should booth exist.

It could also mean that the qemu is not setup correctly and it fails to execute anything.

> If you have a better suggestion for autobuilding for ARM, let me know!

I build my multiarch images with ``docker run --rm --privileged multiarch/qemu-user-static --reset -p yes`` and builx on travis. This makes debugging easier cause it works the exact same way locally. Most of the stuff is in this Makefile https://github.com/SuperSandro2000/docker-images/blob/master/buildx/Makefile.